gallery/src/pageConfig.ts

```diff
@@ -179,6 +180,7 @@ const pageConfig: PageConfig = {
     NxTabs: NxTabsPage,
     NxStatefulTabs: NxStatefulTabsPage,
     NxTag: NxTagPage,
+    NxDateInputPage: NxDateInputPage,
```

Comment:
```suggestion
    NxDateInput: NxDateInputPage,
```

Reply:
> 

This was fixed: it was pushed before I saw this comment: 24e23be4066769cd4e23de1fe5754dbcdace9c05